cypress/e2e/changing-colors.spec.js

```diff
@@ -0,0 +1,61 @@
+/// <reference types="cypress" />
+
+describe('Changing color of the app', ()=>{
+    const colors = {
+      "Light": "rgb(255, 255, 255)",
+      "Dark": "rgb(34, 43, 69)",
+      "Cosmic": "rgb(50, 50, 89)",
+      "Corporate": "rgb(255, 255, 255)"
+     };
+  
+    beforeEach(()=>{
+      cy.visit('/')
+    })
+   /* it('Change color by clicking', () => {
+        cy.wrap(Object.entries(colors)).each(color => {
+          cy.get('ngx-header nb-select button.select-button').click()
+          cy.contains('nb-option', `${color}`).then(color => cy.wrap(Object.entries(colors)).each(rgb => {
+            
+          }))
+          cy.get('nb-layout-header nav').should('have.css', 'background-color', colors.color)
+        })
+      })
+    */
+      it('Switching Light color', () => {
+        cy.wrap(Object.keys(colors))
```

Comment:
Цей код не виконує ніякої дії, можливо ти хотіла створити цикл?
```suggestion
        cy.wrap(Object.keys(colors)).each((color) => {
```